src/validation/index.js

```diff
@@ -38,9 +39,22 @@ const validator = {
       new Check({ description: req }).str().req().min(5);
       new Check({ address: req }).str().req().min(5);
       new Check({ availableSpace: req }).str().req().min(5);
-      new Check({ cost: req }).req().num();
+      new Check({ cost: req }).req().double();
+      new Check({ currency: req }).str().eql(3);
       new Check({ services: req }).str().req().min(5);
       new Check({ amenities: req }).str().req().min(5);
+      req.body.cost = -(0 - (req.body.cost));
+      if (req.body.currency) {
+        let found = false;
+        Object.keys(currencies).forEach(element => {
+          if (element === req.body.currency.toLocaleUpperCase()) {
+            found = true;
```

Comment:
You can use [`indexOf`](https://www.w3schools.com/jsref/jsref_indexof_array.asp) to check if the currency exists.